src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/JwtToStsConverterSubPart.kt

```diff
@@ -78,22 +76,18 @@ class JwtToStsConverterSubPart {
                 }
 
                 parent.modifyResource(it, "Changed targetPort to point to clinger")
-            } else if (it.resource.kind == "Secret") {
-                // map clinger_username/password file
             }
         }
     }
 
-    internal fun createClingerProxyContainer(adc: AuroraDeploymentSpec, context: FeatureContext): Container {
+    internal fun createClingerProxyContainer(adc: AuroraDeploymentSpec, imageMetadata: ImageMetadata): Container {
         val containerPorts = mapOf(
             "http" to PortNumbers.CLINGER_PROXY_SERVER_PORT,
             "management" to PortNumbers.CLINGER_MANAGEMENT_SERVER_PORT
         )
 
-        val imageMetadata = context.imageMetadata
-
         return newContainer {
-            name = "${adc.name}-clinger-mix" // Not naming it sidecar, as that will mask out secrets
+            name = "${adc.name}-clinger-sidecar"
```

Comment:
Jeg antar at renaming til -sidecar har blitt testet, altså at du starter opp en applikasjon med clinger sidecar, og deretter sjekker med `kubectl exec -it ....-HASH -c clinger-sidecar -- env` at miljøvariabel-hemmeligheter ikke har blitt maskert vekk. Om de fremdeles skulle behøves...

Reply:
Det er korrekt. Vi setter opp de aktuelle miljøvariabler manuelt her under. 

Spesifikt hindrer navnet -sidecar at secrets blir satt automatisk fra vault. Dem mapper vi manuelt i denne koden. 

Tilsvarende hindrer navnet -mix at tjenesten blir oppdatert til den konfigurerte versjon (den blir misforstått som "main container" i oversettelsen til deploymentconfig). Det er vesentlig vanskeligere å jobbe seg rundt enn å sette noen verdier :)

Koden er testet i utv01 og fungerer (setter korrekte variable, lar dem bli lest av clinger på rett måte, etc).